compiler/damlc/tests/daml-test-files/LetKeys.daml

```diff
@@ -0,0 +1,12 @@
+daml 1.2
+module LetKeys where
+
+template Foo with
+    operator: Party
+    sigs: [Party]
+    id: Text
+  where
+    signatory sigs
+    key (operator, id): (Party, Text)
+    maintainer key._1
+    let f p = p :: sigs
```

Comment:
We should add a call site of `f` somewhere to make sure the definition doesn't just get dropped.